sdks/python/apache_beam/transforms/create_test.py

```diff
@@ -121,6 +123,43 @@ def test_create_source_progress(self):
 
     self.assertEqual(expected_split_points_report, split_points_report)
 
+  def test_create_uses_coder_for_pickling(self):
+    coders.registry.register_coder(_Unpicklable, _UnpicklableCoder)
+    create = Create([_Unpicklable(1), _Unpicklable(2), _Unpicklable(3)])
+    unpickled_create = pickler.loads(pickler.dumps(create))
+    self.assertEqual(
+        sorted(create.values, key=lambda v: v.value),
+        sorted(unpickled_create.values, key=lambda v: v.value))
+
+    with self.assertRaises(NotImplementedError):
+      create_unpicklable = Create([_Unpicklable(1), 2])
```

Comment:
Why is this unpicklable? I don't understand why _UnpicklableCoder isn't used here as well. Is this using FastPrimitivesCoder instead on a Union[_Unpicklable, int]?

In any case, please add a comment explaining the difference.

Reply:
Unfortunately there is no special 'UnionCoder' as of now, so this falls back to FastPrimitivesCoder. Added a comment.